stats.go

```diff
@@ -107,3 +128,39 @@ func (s *statsExporter) submitMetric(v *view.View, row *view.Row, metricName str
 		return fmt.Errorf("aggregation %T is not supported", v.Aggregation)
 	}
 }
+// calculatePercentile return value for a given percentile using distribution buckets
```

Comment:
I think it's enough to say:
```go
// calculatePercentile returns the percentile value using the specified distribution buckets bounds and
// the number of data points for each bucket represented by countPerBucket.
//
// Buckets and countPerBucket are used to compute the value for a specific percentile.
// Caveat: the precision of the value return depend on the the granularity of the buckets.
```
Usually in Go documentation if some words match the arguments of the function, it is considered that they are referring to them. So this should be a bit better.

Reply:
Sure, let's be concise then ;)